src/classmentors/components/cohorts/cohorts.js

```diff
@@ -583,6 +582,7 @@ function EditCohortCtrl(initialData, spfNavBarService, urlFor, spfAlert, clmData
     this.querySearch = querySearch;
 
     function querySearch (query) {
+        console.log(self.mappedEvents);
```

Comment:
Yes, this is not meant to be permanent. Apologies. We will exclude console calls in future PRs